doc/integrator/upgrade_application.rst.tmpl

```diff
@@ -8,6 +8,10 @@ Upgrading a GeoMapFish application
 Prepare upgrade
 ~~~~~~~~~~~~~~~
 
+Have a look at all the ``Information to know before starting the upgrade`` chapters in the
+`changelog <https://github.com/camptocamp/c2cgeoportal/blob/${MAIN_BRANCH}/geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl>`_.
+If you comes from an LTS you should read all the chapters, otherwise only the first one.
```

Comment:
Suggestion:
"If you are currently using an LTS (Long-term support) release, you should read all the chapters, if you are using the most recent version, only the first chapter is relevant."